TEMPLATE/etc/httpd/conf.d/manageiq-remote-user-openidc.conf

```diff
@@ -10,4 +10,5 @@ RequestHeader set X_REMOTE_USER_FIRSTNAME %{OIDC_CLAIM_GIVEN_NAME}e
 RequestHeader set X_REMOTE_USER_LASTNAME  %{OIDC_CLAIM_FAMILY_NAME}e             env=OIDC_CLAIM_FAMILY_NAME
 RequestHeader set X_REMOTE_USER_FULLNAME  %{OIDC_CLAIM_NAME}e                    env=OIDC_CLAIM_NAME
 RequestHeader set X_REMOTE_USER_GROUPS    %{OIDC_CLAIM_GROUPS}e                  env=OIDC_CLAIM_GROUPS
+RequestHeader set X_REMOTE_USER_DELIMITER ,                                      env=OIDC_CLAIM_GROUPS
```

Comment:
Not sure if this is the right place for this, but `X_REMOTE_USER_DELIMITER` feels like a confusing name, because it should like this delimeter is for the users themselves, and not sure groups?  I'm thinking `X_REMOTE_USER_GROUPS_DELIMITER` and it happens to match name-wise with `X_REMOTE_USER_GROUPS` showing they are related.